### DISCUSSION TITLE
Access to the contents of the binary file (.OUT)

### DISCUSSION THREAD

**Discussion by zsbeheshti** [pyswmm/pyswmm#389] [Q&A] [answered]

**Comment by zsbeheshti** [comment]
Dear all,

In the process of successive SWMM runs in Python environment, I need to access the binary content of the “.OUT” file to extract the results of time series. Does anyone have any related experience to share with me? How can I read this binary content from within Python?

 

Regards

**Comment by bemcdonnell** [comment, marked as answer]
@zsbeheshti,

check out Chapter 4.1 in the pyswmm video tutorial series. This video will show you how to access info from the output file (and takes it a step further for plotting). 

 https://youtu.be/uMF1vSbF_tk

**Reply by bemcdonnell** [reply]
@zsbeheshti Can you provide the additional information?
